### ISSUE TITLE
can't get the real value  from a filed that contain a formula , it return the formula any idea ? 

### ISSUE THREAD

**Issue by samih93** [justkawal/excel#206]

**Comment by JoshH775** [comment]
Did you manage to get around this?

**Comment by FauconSpartiate** [comment]
Formulas aren't supported yet, it's a WIP.
We'll keep you updated when the update drops.

**Comment by samih93** [comment]
Okay thanks

**Comment by samih93** [comment]
> Formulas aren't supported yet, it's a WIP.
> We'll keep you updated when the update drops.

👍

**Comment by XianZhengquan** [comment]
Is there any progress currently?

**Comment by FauconSpartiate** [comment]
No, there are currently no active developers on this project. If you really need this feature, you can try to implement it yourself and open a PR

**Comment by pdalfarr** [comment]
Hi There.

Did someone tried to implement this?
If not, can someone suggest an alternative to this library?

Thanks

**Comment by ibrahim135** [comment]
Hi,
Any update on this guys? Do we have any workaround?

**Comment by pdalfarr** [comment]
No solution on my side
